src/com/projectkorra/projectkorra/firebending/combo/FireWheel.java

```diff
@@ -111,9 +115,6 @@ public void progress() {
 			return;
 		} else if (topBlock.getType() == Material.FIRE) {
 			topBlock = topBlock.getRelative(BlockFace.DOWN);
-		} else if (ElementalAbility.isPlant(topBlock)) {
```

Comment:
As stated by a few other people, this would be better suited as a config option as opposed to simply removing the feature.